example/src/main/java/com/stripe/example/activity/BasePaymentSheetActivity.kt

```diff
@@ -7,8 +7,8 @@ import android.view.MenuItem
 import androidx.activity.viewModels
```

Comment:
Just globalling in the project do a search everywhere for PaymentIntent and see if this is still the correct term.  Here are some examples where I think we still need some updates:
- LaunchPaymentSheetCompleteActivity#createPaymentIntent
- PaymentOptionsViewModel#_paymentItent
- there might be others

Reply:
Yes, there are still a lot of changes to come. I'll do that.